runtime/queries/jq/locals.scm

```diff
@@ -0,0 +1,12 @@
+;; From nvim-treesitter, contributed by @ObserverOfTime et al.
+
+(funcdef
+  (identifier) @local.definition.function)
+
+(funcdefargs
+  (identifier) @local.definition.parameter)
```

Comment:
```suggestion
  (identifier) @local.definition)

(funcdefargs
  (identifier) @local.definition)
```

For locals we only support `local.scope`, `local.definition` and `local.reference` captures